src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+const galleryImg = document.getElementById('largeImg');
+
+gallery.addEventListener('click', (evt) => {
+  evt.preventDefault();
+});
```

Comment:
The event listener on line 6 is redundant because it only prevents the default action without any other functionality. The event listener on line 10 already handles the click event and includes a check for the target element. Consider removing the event listener on line 6 to avoid unnecessary code.